src/main/java/lottery/domain/LotteryResult.java

```diff
@@ -0,0 +1,76 @@
+package lottery.domain;
+
+import java.util.*;
+
+public class LotteryResult {
+    public static final int LIMIT_MATCHED_NUMBER = 3;
+    Map<Integer, Integer> lotteryResultMap;
+    private final Map<Integer, Integer> lotteryValueMap;
+
+    public LotteryResult() {
+        lotteryResultMap = new HashMap<>();
+        lotteryValueMap = new HashMap<Integer, Integer>() {
+            {
+                put(3, 5000);
+                put(4, 50000);
+                put(5, 1500000);
+                put(6, 2000000000);
```

Comment:
Java 7 버전 부터 숫자 자료형의 가독성을 높일수 있도록 언더스코어를 사용할수 있게 되었습니다. 
```suggestion
                put(6, 2_000_000_000);
```

Reply:
아네 언더스코어 사용은 제가 몰랐던 부분입니다.
피드백 감사합니다. :)